apps/settings/js/bluetooth.js

```diff
@@ -685,18 +688,25 @@ navigator.mozL10n.once(function bluetoothSettings() {
     }
 
     function setDeviceUnpair(device) {
+      function unpairDevice() {
+        // backend takes responsibility to disconnect first.
+        var req = defaultAdapter.unpair(device.address);
+        req.onerror = function bt_pairError() {
+          showDevicePaired(true, null);
+        };
+      }
+
       if (device.address === connectedAddress) {
-        var msg = _('unpair-title') + '\n' + _('unpair-msg');
-        if (!window.confirm(msg)) {
-          return;
-        }
-        connectedAddress = null;
+        navigator.mozL10n.formatValue('unpair-confirm').then(msg => {
+          if (!window.confirm(msg)) {
+            return;
+          }
```

Comment:
Similarly as above, I can't seem to be able to trigger this codepath. All I'm getting is using this: https://github.com/mozilla-b2g/gaia/blob/master/apps/settings/js/panels/bluetooth/panel.js#L517-L522

Can you give me STR to launch this codepath?